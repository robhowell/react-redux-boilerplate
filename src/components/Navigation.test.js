import React from 'react';
import { shallow } from 'enzyme';
import Navigation from './Navigation';

describe('Components/Navigation Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Navigation />);
    expect(wrapper).toMatchSnapshot();
  });
});
