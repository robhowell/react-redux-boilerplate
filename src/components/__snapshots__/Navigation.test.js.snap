// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Navigation Component should render correctly 1`] = `
<ul>
  <li>
    <ForwardRef
      to="/"
    >
      Home
    </ForwardRef>
  </li>
  <li>
    <ForwardRef
      to="/about"
    >
      About
    </ForwardRef>
  </li>
</ul>
`;
